src/components/ButtonFirebase/index.js

```diff
@@ -24,6 +24,36 @@ const sendToBackendToken = () => {
   })
 }
 
+const firebaseAuth = () => () => {
+  firebase.auth().currentUser.getIdToken(/* forceRefresh */ true).then(function (idToken) {
```

Comment:
This code, can be part of a service in other file

Reply:
Thanks for the code review Hector, I changed it :)